components/family-studies/ui/src/main/resources/PhenoTips/FamilyRecordField_pedigree.xml

```diff
@@ -117,7 +117,7 @@
   #end
 #end
 #if ("$!{pedigreeHtml}" != '')
-  (% id='pedigree' class='chapter clinical-info pedigree-family-section' %)(((
+  (% id='pedigree' class='clinical-info pedigree-family-section' %)(((
```

Comment:
This causes a merge conflict because ` FamilyRecordField_pedigree.xml` has been changed. Please update.